### ISSUE TITLE
Querying SDL Raises Exception in 1.0.1

### ISSUE THREAD

**Issue by rtymchyk** [Gusto/apollo-federation-ruby#51]
The error is `unknown keywords: request, current_user, auth, __pro_access_strategy__`

Reported from:
```
/gem_bag/ruby/2.5.0/gems/apollo-federation-1.0.1/lib/apollo-federation/schema.rb:50:in `federation_sdl'
/gem_bag/ruby/2.5.0/gems/apollo-federation-1.0.1/lib/apollo-federation/service_field.rb:13:in `_service'
```

The query is:
```
query GetServiceDefinition { _service { sdl } }
```

Which is fired automatically from `@apollo/gateway`.

Fairly certain it is from https://github.com/Gusto/apollo-federation-ruby/pull/45.

**Comment by rtymchyk** [comment]
The problem turned out to be that you need `graphql >= 1.9.18` for `apollo-federation >= 1.0.1`. We could update the dependencies but not sure if that's worth the hassle.

**Comment by rtymchyk** [comment]
Apologies for the rollercoaster of updates here. At this point I am on `graphql = 1.9.19` and `apollo-federation = 1.0.1`, yet I am still seeing an exception thrown.

**Comment by VinSpee** [comment]
I'm similarly getting `unknown keyword: controller` on 

```
Processing by GraphqlController#execute as */*
  Parameters: {"query"=>"query GetServiceDefinition { _service { sdl } }", "graphql"=>{"query"=>"query GetServiceDefinition { _service { sdl } }"}}
```

**Comment by robinboening** [comment]
Any update on this? I am running into the same issue atm.

**Comment by rtymchyk** [comment]
This is fixed in v1.0.2

**Comment by robinboening** [comment]
Nice 👍 thanks!
